### ISSUE TITLE
Use standard Windows font

### ISSUE THREAD

**Issue by mrwee** [Grovkillen/ESP_Easy_Flasher#7]
I really like this tool, but sometimes find the text a bit difficult to read. May I propose to use the standard Windows font, so the flasher adapts to the user's preferences?

**Comment by Grovkillen** [comment]
I'm gonna release an official tool later this autumn so no much time is given to this app for that reason.

**Comment by Grovkillen** [comment]
Okay I might consider continuing on this app :) I'm considering the fact that I might do an overhaul of the GUI + letting the user tweak it a bit :)

**Comment by localhost61** [comment]
I confirm that it's surprising ugly, but it looks better once the fonts or manually copied into the OS font folder as requested by the highlighted message in the application working area.
Here is what I first get on my Laptop (FHD resolution) with 0.4.7
![espeasyflasher](https://user-images.githubusercontent.com/33861984/53486942-6ec7da80-3a8a-11e9-9071-a07a83ebdd27.png)
And after restart when the fonts are installed
![espeasyflasher_2](https://user-images.githubusercontent.com/33861984/53487783-c23b2800-3a8c-11e9-9abf-d59b5443e087.png)

**Comment by mrwee** [comment]
Yes, I've also found that installing the fonts, makes it better. However I'm not that crazy about the font itself, which was why I asked for the application to use the system font instead. That would be an great user option.

**Comment by Grovkillen** [comment]
I will most likely turn it into a NodeJS app for multi OS support.

Den ons 27 feb. 2019 15:28mrwee <notifications@github.com> skrev:

**Comment by Grovkillen** [comment]
https://www.patreon.com/posts/46356158
